packages/apps/desktop-viewer-test/src/frontend/components/App.tsx

```diff
@@ -0,0 +1,36 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useDesktopViewerInitializer } from "@itwin/desktop-viewer-react";
+import { Router } from "@reach/router";
+import React from "react";
+
+import {
+  DownloadRoute,
+  HomeRoute,
+  IModelsRoute,
+  ITwinsRoute,
+  SnapshotRoute,
+} from "./routes";
+
+const App = () => {
+  const initialized = useDesktopViewerInitializer();
+
+  return initialized ? (
+    <div style={{ height: "100%" }}>
+      <Router style={{ height: "calc(100% - 20px)" }}>
+        <HomeRoute path="/" />
+        <DownloadRoute path="itwins/:iTwinId/:iModelId" />
```

Comment:
no leading `/` ?

Reply:
by design.....so it isnt made a child of the current route (there are nested routes in reach router)